oauth-server/src/main/java/io/strimzi/kafka/oauth/server/ServerConfig.java

```diff
@@ -83,6 +83,11 @@ public class ServerConfig extends Config {
      */
     public static final String OAUTH_GROUPS_CLAIM_DELIMITER = "oauth.groups.claim.delimiter";
 
+    /**
+     * "oauth.include.accept.header"
+     */
+    public static final String OAUTH_INCLUDE_ACCEPT_HEADER = "oauth.include.accept.header";
```

Comment:
Should there be some documentation for this option?

Reply:
This is just a string constant in `ServerConfig.java`. Following the existing established pattern the option should be documented in `JaasServerOauthValidatorCallbackHandler.java` and in `README.md`.
But actually, I see this as a global option which should be moved from `ServerConfig.java` to `Config.java`, and taken into account also in `KeycloakRBACAuthorizer.java`, and in `JaasClientOauthLoginCallbackHandler`.